Windows build is failing https://github.com/Thar0/pygfxd/actions/runs/13215163829/job/36893557703?pr=16
```
Run pipx run --python "C:\hostedtoolcache\windows\Python\3.11.9\x64\python.exe" --spec "D:\a\_actions\pypa\cibuildwheel\v2.16.2" cibuildwheel "." --output-dir '"wheelhouse"' --config-file '""' --only '""'
  pipx run --python "C:\hostedtoolcache\windows\Python\3.11.9\x64\python.exe" --spec "D:\a\_actions\pypa\cibuildwheel\v2.16.2" cibuildwheel "." --output-dir '"wheelhouse"' --config-file '""' --only '""'
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
  env:
    CIBW_ARCHS_WINDOWS: auto
    CIBW_ARCHS_LINUX: auto
    CIBW_ARCHS_MACOS: all
Found a space in the pipx home path. We heavily discourage this, due to
    multiple incompatibilities. Please check our docs for more information on
    this, as well as some pointers on how to migrate to a different home path.
creating virtual environment...
determining package name from 'D:\\a\\_actions\\pypa\\cibuildwheel\\v2.16.2'...
creating virtual environment...
installing cibuildwheel from spec 'D:\\a\\_actions\\pypa\\cibuildwheel\\v2.16.2'...
Invalid --only='""', must be a build selector with a known platform
Error: Process completed with exit code 1.
```
I found this issue which seems relevant https://github.com/pypa/cibuildwheel/issues/1740
I'll try upgrading the `cibuildwheel` action version as well

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -24,7 +24,7 @@ jobs:
 
       - uses: actions/upload-artifact@v4
         with:
-          name: artifact-wheel-${{ matrix.os }}
+          name: artifact-wheels-${{ matrix.os }}
           path: ./wheelhouse/*.whl
 
   build_sdist:

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.16.2
+        uses: pypa/cibuildwheel@v2.22.0
         env:
           CIBW_ARCHS_WINDOWS: "auto"
           CIBW_ARCHS_LINUX: "auto"

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -24,6 +24,7 @@ jobs:
 
       - uses: actions/upload-artifact@v4
         with:
+          name: artifact-wheel-${{ matrix.os }}
           path: ./wheelhouse/*.whl
 
   build_sdist:
@@ -37,6 +38,7 @@ jobs:
 
       - uses: actions/upload-artifact@v4
         with:
+          name: artifact-sdist
           path: dist/*.tar.gz
 
   upload_pypi:
@@ -46,7 +48,7 @@ jobs:
     steps:
       - uses: actions/download-artifact@v4
         with:
-          name: artifact
+          pattern: artifact-*
           path: dist
 
       - uses: pypa/gh-action-pypi-publish@v1.8.11

